policyenforcer.go

```diff
@@ -46,19 +46,19 @@ type ValidationReport struct {
 
 // PolicyEnforcer is an interface with methods that make policy decisions.
 type PolicyEnforcer interface {
-	// Evaluate determines the final outcome of validation that is constructed 
+	// Evaluate determines the final outcome of validation that is constructed
 	// using the results from individual verifications.
-	Evaluate(ctx context.Context, artifactReports []*ValidationReport) bool
+	Evaluate(ctx context.Context, artifactReports []*ValidationReport) (bool, error)
```

Comment:
reading the comments, its unclear to me how validationReport is different to  VerificationResult. Why do we only evaluate the report, not the result :)  Could we add more details to the comments? thanks